marketplace/deployer_util/deploy.sh

```diff
@@ -69,6 +69,28 @@ create_manifests.sh
   --manifests "/data/manifest-expanded" \
   --dest "/data/resources.yaml"
 
+# Kubeflow hack: Remove the owner reference on cluster-scoped IAM resources.
+if kubectl auth can-i list,patch clusterroles \
```

Comment:
I think it's safer to also do this for service account. I don't know if KSA is deleted by deployer image is a deterministic behavior please ignore if not the case.

Reply:
ServiceAccount is namespaced so there is no risk with it being owned by the application.

Currently, we delete the deployer KSA, Role, and RoleBinding in one line which is nonideal, but a separate issue. Instead we should only delete the deployer KSA, and have it own the Role and RoleBinding. I've added this comment to https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/issues/347